docs/ENVIRONMENT_VARIABLES.md

```diff
@@ -14,6 +14,8 @@ in the different contexts.
   is required.
 * `PCS_STORAGEADAPTER_WEBSERVICE_URL` [mandatory]: the URL where the storage
   adapter service is available, e.g. `http://127.0.0.1:9022/v1`.
+* `PCS_AZURE_STORAGE_ACCOUNT` [mandatory]: contains the full connection string
```

Comment:
need to add as PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING well

Reply:
Fixed.